src/Pages/Detail/DetailMain.js

```diff
@@ -1,115 +1,119 @@
-import React, { Component } from "react";
-import "./DetailMain.scss";
-import Color from "./Color";
+import React, { Component } from 'react';
+import './DetailMain.scss';
+import ColorList from './ColorList';
 
 class DetailMain extends Component {
-  constructor() {
-    super();
-    this.state = {
-      count: 1,
-      totalPrice: 19000,
-      colorList: [
-        { title: "얼루어먼트", bgc: "#C48490" },
-        { title: "오운 스킨", bgc: "#E8927C" },
-        { title: "포인트 오브 뷰", bgc: "#E03C31" },
-        { title: "언체인드", bgc: "#EBA0A5" },
-        { title: "해치 아웃", bgc: "#E3785E" },
-        { title: "파워하우스", bgc: "#DF4661" },
-        { title: "스탠스", bgc: "#BE3A34" },
-        { title: "뉴 퍼스펙티브", bgc: "#C74d66" },
-        { title: "오픈 도어", bgc: "#D05A57" },
-        { title: "풀 크레딧", bgc: "#BA0C2F" },
-      ],
-    };
-  }
-
-  plusOne = () => {
-    if (this.state.count >= 5) {
-      alert("최대 주문수량은 5개 입니다.");
-      return;
+    constructor(props) {
+        super(props);
+        this.state ={
+            count: 1,
+            totalPrice: null,
+            colorList :[]
+        }
     }
-    this.setState({
-      count: this.state.count + 1,
-      totalPrice: this.state.totalPrice + 19000,
-    });
-  };
 
-  minusOne = () => {
-    if (this.state.count <= 1) {
-      alert("최소 주문수량은 1개 입니다");
-      return;
+
+    componentDidMount =() => {
+        console.log(this.state.colorList)
+        fetch('http://localhost:3000/data/data.json')
```

Comment:
```suggestion
    componentDidMount() {
        fetch('/data/data.json')
```
- 라이프사이클 메서드는 화살표 함수로 쓰지 않습니다
- 테스트 끝난 console 지워주세요